test/integration/tasks.test.js

```diff
@@ -112,6 +112,33 @@ describe("Tasks", function () {
           return done();
         });
     });
+    it('Should return fail response after adding the task with a non-acceptable value for the "status" in the task object ', function (done) {
+      chai
+        .request(app)
+        .post("/tasks")
+        .set("cookie", `${cookieName}=${jwt}`)
+        .send({
+          title: "Test task - Create",
+          type: "feature",
+          endsOn: 123,
+          startedOn: 456,
+          status: "Helloworld",
```

Comment:
I think it will be more readable with `status` being `invalidStatus` or `dummyStatus`. `Helloworld` status is confusing and does'nt help in readablity

Reply:
Sure